Servers/UI/OJS.Servers.Ui/Controllers/AccountController.cs

```diff
@@ -44,20 +56,35 @@ public async Task<IActionResult> Login([FromBody]LoginRequestModel model)
                 return this.RedirectToAction("Index", "Home");
             }
 
-            var signInResult = await this.signInManager
-                .PasswordSignInAsync(model.UserName, model.Password, model.RememberMe, false);
+            ExternalUserInfoModel? externalUser;
+
+            var platformCallResult = await this.sulsPlatformHttpClient.GetAsync<ExternalUserInfoModel>(
+                new { model.UserName },
+                string.Format(GetUserInfoByUsernamePath));
 
-            if (!signInResult.Succeeded)
+            if (platformCallResult.IsSuccess)
```

Comment:
why not merge the checks and remove the else if

```
if (!platformCallResult.IsSuccess && EnvironmentUtils.IsProduction())
{
    return this.Unauthorized(GlobalConstants.ErrorMessages.InactiveLoginSystem);
}
```

..rest of code outside if blocks